payment_wcpay/models/weixinsdk.py

```diff
@@ -0,0 +1,607 @@
+#coding:utf-8
+"""
```

Comment:
can't we add it as dependency instead of including the file here. It is probably OK but I think we can improve this.
Besides the file is quite old: I am quite sure it is possible to get a newer file